From bf9c7ad76c74186744a6aa7f02fd3bcd74dc7dd3 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Thu, 4 Jul 2024 03:30:20 +0000
Subject: [PATCH] WA Disable trusty for MTL BM Boot

This patch will disable the trusty for MTL BM boot

Tests Done: NA

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 caas/mixins.spec | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/caas/mixins.spec b/caas/mixins.spec
index 18312100..b43571ee 100755
--- a/caas/mixins.spec
+++ b/caas/mixins.spec
@@ -9,7 +9,7 @@ product.mk: device.mk
 [groups]
 kernel: gmin64(useprebuilt=false,src_path=kernel/lts2023-chromium, loglevel=7, interactive_governor=false, relative_sleepstates=false, modules_in_bootimg=false, external_modules=,debug_modules=, use_bcmdhd=false, use_iwlwifi=false, extmod_platform=bxt, iwl_defconfig=, cfg_path=config-lts/lts2023-chromium, more_modules=true, lts2021_chromium_src_path=kernel/lts2021-chromium, lts2021_chromium_cfg_path=config-lts/lts2021-chromium, linux_intel_lts2021_src_path=kernel/linux-intel-lts2021, linux_intel_lts2021_cfg_path=config-lts/linux-intel-lts2021, lts2022_chromium_src_path=kernel/lts2022-chromium, lts2022_chromium_cfg_path=config-lts/lts2022-chromium, linux_intel_lts2022_src_path=kernel/linux-intel-lts2022, linux_intel_lts2022_cfg_path=config-lts/linux-intel-lts2022)
 disk-bus: auto
-boot-arch: project-celadon(uefi_arch=x86_64,fastboot=efi,ignore_rsci=true,disable_watchdog=true,watchdog_parameters=10 30,verity_warning=false,txe_bind_root_of_trust=false,bootloader_block_size=4096,verity_mode=false,disk_encryption=false,file_encryption=true,metadata_encryption=true,fsverity=true,target=caas,ignore_not_applicable_reset=true,self_usb_device_mode_protocol=true,usb_storage=true,live_boot=true,userdata_checkpoint=true,data_use_f2fs=true,trusty=true)
+boot-arch: project-celadon(uefi_arch=x86_64,fastboot=efi,ignore_rsci=true,disable_watchdog=true,watchdog_parameters=10 30,verity_warning=false,txe_bind_root_of_trust=false,bootloader_block_size=4096,verity_mode=false,disk_encryption=false,file_encryption=true,metadata_encryption=true,fsverity=true,target=caas,ignore_not_applicable_reset=true,self_usb_device_mode_protocol=true,usb_storage=true,live_boot=true,userdata_checkpoint=true,data_use_f2fs=true)
 sepolicy: enforcing
 bluetooth: btusb(ivi=false)
 audio: project-celadon
@@ -52,14 +52,14 @@ lights: true
 power: true(power_throttle=true)
 debug-usb-config: true(source_dev=dvcith-0-msc0)
 intel_prop: true
-tee: trusty(ref_target=celadon_64)
+tee: false
 memtrack: true
-tpm: true
+tpm: false
 avx: auto
 health: hal
 slot-ab: true
 abota-fw: true
-firststage-mount: true(trusty=true)
+firststage-mount: true
 cpuset: autocores
 usb-init: true
 usb-audio-init: false
-- 
2.34.1

